tmail-backend/apps/distributed-es6-backport/src/test/resources/blob.properties

```diff
@@ -0,0 +1,85 @@
+# ========================================= ObjectStorage deduplication ========================================
```

Comment:
I think this file can be removed

Reply:
If we don't have it, DistributedServerTest/EncryptedDistributedServerTest will throw exception